src/app/containers/Metadata/index.test.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Helmet from 'react-helmet';
+import MetadataContainer from './index';
+import { ServiceContextProvider } from '../../components/ServiceContext';
+import { articleDataNews, articleDataPersian } from '../Article/fixtureData';
+import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
+import services from '../../lib/config/services/index';
+
+const MetadataWithContextAsObject = (service, serviceFixtureData) => {
+  const { metadata, promo } = serviceFixtureData;
+
+  renderer.create(
+    <ServiceContextProvider service={service}>
+      <MetadataContainer
+        isAmp={false}
+        metadata={metadata}
+        promo={promo}
+        service={service}
+      />
+    </ServiceContextProvider>,
+  );
+
+  return Helmet.peek();
+};
+
+describe('no data', () => {
+  shouldShallowMatchSnapshot(
+    'should render null',
+    <MetadataContainer isAmp={false} metadata={{}} promo={{}} service="" />,
+  );
+});
+
+const articleMetadataBuilder = (
```

Comment:
Phil & I feel this is a bit of an unnecessary warning -- we're only returning an object, the line count doesn't feel like a representative metric here. We could create a function that builds the meta tags array, but it would increase the logical complexity & would only be done to appease code climate.